Kernel/Syscalls/mmap.cpp

```diff
@@ -600,7 +600,14 @@ ErrorOr<FlatPtr> Process::sys$msync(Userspace<void*> address, size_t size, int f
         // FIXME: Handle MS_ASYNC
         TRY(inode_vmobject.sync(offset / PAGE_SIZE, rounded_size / PAGE_SIZE));
         // FIXME: Handle MS_INVALIDATE
-        // FIXME: If msync() causes any write to a file, the file's st_ctime and st_mtime fields shall be marked for update.
+
+        // FIXME: This should be handled by vmobject::sync -> Inode::write_bytes -> Inode::did_modify_contents instead
```

Comment:
In that case, do we even want to attempt to handle those here, or do we just move the FIXME to where it actually makes the most sense to handle this?

Reply:
I moved this to the likely correct place